src/extension.ts

```diff
@@ -24,11 +24,26 @@ import {CodelensProvider} from './Editor/CodelensProvider';
 import {HoverProvider} from './Editor/HoverProvider';
 import {Jsontracer} from './Jsontracer';
 import {Project} from './Project';
+import {CompilableFile} from './Support/CompilableFile';
 import {Utils} from './Utils';
 
 export function activate(context: vscode.ExtensionContext) {
   console.log('one-vscode activate OK');
 
+  // Thses context is used to show "Compile" menu in Explorer view
+  //
+  // 1. When a file is right-clicked (e.g., .pb, .tflite, etc)
+  // 2. When a dir is right-clicked (e.g., Keras model or saved model)
+  CompilableFile.setFileExtContext('onevscode.compilableExtList');
```

Comment:
To me, it seems that `CompilableFile` is there and its file ext context is `onevscode.compilableExtList`.